src/main/java/org/mitre/inferno/rest/Endpoints.java

```diff
@@ -16,44 +21,82 @@ public class Endpoints {
   private final Validator validator;
   private final FHIRPathEvaluator pathEvaluator;
 
-  private Endpoints(Validator validator, FHIRPathEvaluator evaluator, int port) {
+  private Endpoints(Validator validator, FHIRPathEvaluator evaluator) {
     this.validator = validator;
     this.pathEvaluator = evaluator;
-    port(port);
     createRoutes();
   }
 
+  /**
+   * Set up temporary routes while the validator still loads.
+   * 
+   * @param port the port to listen for requests on
+   */
+  public static void setupLoadingRoutes(int port) {
+    port(port);
+    before((req, res) -> {
+      res.header("Access-Control-Allow-Origin", "*");
+      res.header("Access-Control-Allow-Methods", "GET, POST, PUT, OPTIONS");
+      res.header("Access-Control-Allow-Headers", "*");
+    });
+
+    // This responds to OPTIONS requests, used by browsers to "preflight" check CORS
+    // requests,
+    // with a 200 OK response with no content and the CORS headers above
+    options("*", (req, res) -> "");
+
+    JsonObject warning = new JsonObject();
+    warning.addProperty("Warning", "Validator still loading... please wait.");
+
```

Comment:
Red errors should indicate a problem with the system under test, while purple errors indicate a problem with Inferno, so this should be a purple error. A 503 seems like a reasonable error code for this, so I think we should return a 503 with the appropriate error message, and then (as a separate ticket) in inferno check for that error code (or any non-success) and render a purple error without the not very helpful stack trace.

Reply:
Sounds good to me, and also sounds like that falls under FI-2035 which I'm currently working on so I'll see if that can be done without major refactoring

Reply:
added 503 status response to be caught and added back the headers method I had removed

Reply:
Now returns Operation Outcome, which is marked purple, but of course the message is not readable. If you uncomment the last line to parse it into a string, the "wait" message appears, but the test is marked red. Since we had wanted a purple test, indicating a validator issue (not a testing/system issue), and an operation outcome return type, I am leaving it commented.

Reply:
I'm making the corresponding change in inferno-core to better handle validation errors (PR to come but see branch `fi-2035-validator-errors`) . It's not really useful to return a string like "org.hl7.fhir.r5.model.OperationOutcome@500783f8" so please uncomment that bit that converts it to a String, and we'll just have to coordinate the pieces going live together